### ISSUE TITLE
Compliance Amount Change History similar to Folder Location Change History

### ISSUE THREAD

**Issue by vatsalbhatia** [madrecha/jamku#873]
### Feature Request

Compliance Amount Change History similar to Folder Location Change History

**Comment by vatsalbhatia** [comment]
It would be more technical but if this is possible this would be best case ...
where we can mention period wise default rates eg

01.04.2020 amount would be 10000
01.04.2021 it would be 15000
01.04.2022 it would be 20000.

however Amount Change History is still needed.

**Comment by adarshmadrecha** [comment]
I agree, there should be option to schedule the change in Compliance Amount as well as history of changes in compliance.
This would be taken up in v4
